src/containers/NoteArea/NoteArea.js

```diff
@@ -22,30 +22,38 @@ export class NoteArea extends Component {
     const noteCards = notes.map(note => {
       const noteItems = items.filter(item => item.noteID === note.id)
 
-      return <NoteCard {...note} noteItems={noteItems} key={note.id}/>
+      return <NoteCard {...note} noteItems={noteItems} key={note.id} />
     })
 
-    return (
-      <Masonry breakpointCols={breakpointColumnsObj}
-        className="note-area-grid"
-        columnClassName="note-area-grid_column">
-        {noteCards}
-      </Masonry>
-    )
+    switch (isLoading) {
+      case true:
+        return <div>...Loading</div>
```

Comment:
This makes sense here. Same as comment on App - perhaps we can turn this into some component in case we have loading somewhere else.

Reply:
Good idea Eric, I'll add an issue so that we can merge this